core-server-kit/autogen.kit.d/templates/sudo.tmpl

```diff
@@ -0,0 +1,254 @@
+# Distributed under the terms of the GNU General Public License v2
+
```

Comment:
Can you add the comment: `Created by MARK Devkit`

This will help on checking what packages are autogen